From 1d92e5e98e3cb86b56ad5b54782f825f6a1d6dd7 Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Wed, 14 Apr 2021 13:24:28 -0500
Subject: [PATCH 083/196] Update testnet chainTxData (#150)

This makes it so that the progress in the log is displayed (somewhat) correctly

Closes #148.
---
 src/chainparams.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/chainparams.cpp b/src/chainparams.cpp
index 65a66073a..cb5ff0c29 100644
--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -308,9 +308,9 @@ public:
         checkpointData = CheckpointData(CBaseChainParams::TESTNET);
 
         // Data as of block
-        // 000000000005b07ecf85563034d13efd81c1a29e47e22b20f4fc6919d5b09cd6
-        // (height 1223263)
-        chainTxData = ChainTxData{1522608381, 15052068, 0.15};
+        // 000000000ecaba087910aaf66ade754e6972f6bbefa3f396514501589602b060
+        // (height 5251)
+        chainTxData = ChainTxData{1618419413, 5260, 0.01};
     }
 };
 
-- 
2.39.5 (Apple Git-154)

